sdk/cosmos/tests.yml

```diff
@@ -251,8 +251,8 @@ jobs:
         TestName: Integration_Test_Spring_Data_Cosmosdb
         ServiceDirectory: cosmos
         Artifacts:
-            - name: spring-data-cosmosdb
-              groupId: com.microsoft.azure
+            - name: azure-spring-data-cosmos
+              groupId: com.azure
```

Comment:
With the artifact changing the safe name would be azurespringdatacosmos. The ci.yml apparently already has this change.

Reply:
Yups, done.